src/js/components/GroupNavDMButtons/GroupNavDMButtons.test.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { mount } from 'enzyme';
+import GroupNavDMButtons from "./GroupNavDMButtons";
+import Config from "../../utilities/Config";
+import { GlobalContext } from "../../contexts/GlobalContextWrapper";
+
+// it displays a “Direct messages” component in the secondary navigation
+// it displays an “add” button
+// it displays a new Direct message placeholder button, when I interact with the “add” button
+// it removes the the DM component item from view, when I interact with the “remove” button
+
+describe('GroupNavDMButtons', () => {
+  const fooConfiguration = {
+    groupNavDMButtons: [
+      {
+        id: 'foo'
+      },
+    ]
+  };
+
+  const fooValues = {
+    state: {
+      currentGroupNavDMButtonId: '',
+    }
+  }
+
+  let fooConfig;
+
+  beforeEach(() => {
+    fooConfig = new Config(fooConfiguration)
+  })
+
+  it("displays a “Direct messages” component in the secondary navigation", () => {
```

Comment:
I'm pretty sure it does, but now Im not sure

Reply:
This does prove that a "DM" component (which I'm interpreting as a "DM menu item") is rendered in the `GroupNavDMButtons`, but it _technically_ doesn't prove that it displays in the secondary nav, because the nav can still render, even if the `GroupNavDMButtons` does not 😅, so I thought I would ask. Reckon I'm being too particular? Or do you think testing an AC like this might belong in the `GroupNav.test.js` set?